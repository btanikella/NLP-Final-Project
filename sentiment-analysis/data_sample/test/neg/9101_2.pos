DT NN VBZ RB JJ TO VB DT  NN  NN  CC PRP VBZ NNP NNP VBZ DT NN IN DT NN NN NN WDT VBZ -LRB-  -RRB-
IN NN TO VB DT JJ NN IN VBG DT JJ NN NN NN : DT VBG : DT JJ NNS : NN NN NN CD

RB DT JJ NN CD

