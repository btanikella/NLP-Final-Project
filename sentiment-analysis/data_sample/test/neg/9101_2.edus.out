Sentence #1 (31 tokens):
This film tries very hard to be an "action" film, but it fails miserably.Steve Guttenberg plays the head of an elite counter-terrorist team that fails (?)
[Text=This CharacterOffsetBegin=4 CharacterOffsetEnd=8 PartOfSpeech=DT] [Text=film CharacterOffsetBegin=9 CharacterOffsetEnd=13 PartOfSpeech=NN] [Text=tries CharacterOffsetBegin=14 CharacterOffsetEnd=19 PartOfSpeech=VBZ] [Text=very CharacterOffsetBegin=20 CharacterOffsetEnd=24 PartOfSpeech=RB] [Text=hard CharacterOffsetBegin=25 CharacterOffsetEnd=29 PartOfSpeech=JJ] [Text=to CharacterOffsetBegin=30 CharacterOffsetEnd=32 PartOfSpeech=TO] [Text=be CharacterOffsetBegin=33 CharacterOffsetEnd=35 PartOfSpeech=VB] [Text=an CharacterOffsetBegin=36 CharacterOffsetEnd=38 PartOfSpeech=DT] [Text=`` CharacterOffsetBegin=39 CharacterOffsetEnd=40 PartOfSpeech=``] [Text=action CharacterOffsetBegin=40 CharacterOffsetEnd=46 PartOfSpeech=NN] [Text='' CharacterOffsetBegin=46 CharacterOffsetEnd=47 PartOfSpeech=''] [Text=film CharacterOffsetBegin=48 CharacterOffsetEnd=52 PartOfSpeech=NN] [Text=, CharacterOffsetBegin=52 CharacterOffsetEnd=53 PartOfSpeech=,] [Text=but CharacterOffsetBegin=54 CharacterOffsetEnd=57 PartOfSpeech=CC] [Text=it CharacterOffsetBegin=58 CharacterOffsetEnd=60 PartOfSpeech=PRP] [Text=fails CharacterOffsetBegin=61 CharacterOffsetEnd=66 PartOfSpeech=VBZ] [Text=miserably.Steve CharacterOffsetBegin=67 CharacterOffsetEnd=82 PartOfSpeech=NNP] [Text=Guttenberg CharacterOffsetBegin=83 CharacterOffsetEnd=93 PartOfSpeech=NNP] [Text=plays CharacterOffsetBegin=94 CharacterOffsetEnd=99 PartOfSpeech=VBZ] [Text=the CharacterOffsetBegin=100 CharacterOffsetEnd=103 PartOfSpeech=DT] [Text=head CharacterOffsetBegin=104 CharacterOffsetEnd=108 PartOfSpeech=NN] [Text=of CharacterOffsetBegin=109 CharacterOffsetEnd=111 PartOfSpeech=IN] [Text=an CharacterOffsetBegin=112 CharacterOffsetEnd=114 PartOfSpeech=DT] [Text=elite CharacterOffsetBegin=115 CharacterOffsetEnd=120 PartOfSpeech=NN] [Text=counter-terrorist CharacterOffsetBegin=121 CharacterOffsetEnd=138 PartOfSpeech=NN] [Text=team CharacterOffsetBegin=139 CharacterOffsetEnd=143 PartOfSpeech=NN] [Text=that CharacterOffsetBegin=144 CharacterOffsetEnd=148 PartOfSpeech=WDT] [Text=fails CharacterOffsetBegin=149 CharacterOffsetEnd=154 PartOfSpeech=VBZ] [Text=-LRB- CharacterOffsetBegin=155 CharacterOffsetEnd=156 PartOfSpeech=-LRB-] [Text=? CharacterOffsetBegin=156 CharacterOffsetEnd=157 PartOfSpeech=.] [Text=-RRB- CharacterOffsetBegin=157 CharacterOffsetEnd=158 PartOfSpeech=-RRB-] 
root(ROOT-0, tries-3)
det(film-2, This-1)
nsubj(tries-3, film-2)
advmod(hard-5, very-4)
advmod(tries-3, hard-5)
aux(film-12, to-6)
cop(film-12, be-7)
det(film-12, an-8)
punct(film-12, ``-9)
nn(film-12, action-10)
punct(film-12, ''-11)
xcomp(tries-3, film-12)
punct(tries-3, ,-13)
cc(tries-3, but-14)
nsubj(fails-16, it-15)
conj(tries-3, fails-16)
nn(Guttenberg-18, miserably.Steve-17)
nsubj(plays-19, Guttenberg-18)
ccomp(fails-16, plays-19)
det(head-21, the-20)
dobj(plays-19, head-21)
prep(head-21, of-22)
det(team-26, an-23)
nn(team-26, elite-24)
nn(team-26, counter-terrorist-25)
pobj(of-22, team-26)
nsubj(fails-28, that-27)
rcmod(team-26, fails-28)
punct(fails-28, -LRB--29)
punct(tries-3, ?-30)
punct(tries-3, -RRB--31)

Sentence #2 (26 tokens):
in attempt to keep a mysterious group from stealing a deadly nerve agent.The story...the acting...the special effects...ALL FALL FLAT!!
[Text=in CharacterOffsetBegin=163 CharacterOffsetEnd=165 PartOfSpeech=IN] [Text=attempt CharacterOffsetBegin=166 CharacterOffsetEnd=173 PartOfSpeech=NN] [Text=to CharacterOffsetBegin=174 CharacterOffsetEnd=176 PartOfSpeech=TO] [Text=keep CharacterOffsetBegin=177 CharacterOffsetEnd=181 PartOfSpeech=VB] [Text=a CharacterOffsetBegin=182 CharacterOffsetEnd=183 PartOfSpeech=DT] [Text=mysterious CharacterOffsetBegin=184 CharacterOffsetEnd=194 PartOfSpeech=JJ] [Text=group CharacterOffsetBegin=195 CharacterOffsetEnd=200 PartOfSpeech=NN] [Text=from CharacterOffsetBegin=201 CharacterOffsetEnd=205 PartOfSpeech=IN] [Text=stealing CharacterOffsetBegin=206 CharacterOffsetEnd=214 PartOfSpeech=VBG] [Text=a CharacterOffsetBegin=215 CharacterOffsetEnd=216 PartOfSpeech=DT] [Text=deadly CharacterOffsetBegin=217 CharacterOffsetEnd=223 PartOfSpeech=JJ] [Text=nerve CharacterOffsetBegin=224 CharacterOffsetEnd=229 PartOfSpeech=NN] [Text=agent.The CharacterOffsetBegin=230 CharacterOffsetEnd=239 PartOfSpeech=NN] [Text=story CharacterOffsetBegin=240 CharacterOffsetEnd=245 PartOfSpeech=NN] [Text=... CharacterOffsetBegin=245 CharacterOffsetEnd=248 PartOfSpeech=:] [Text=the CharacterOffsetBegin=248 CharacterOffsetEnd=251 PartOfSpeech=DT] [Text=acting CharacterOffsetBegin=252 CharacterOffsetEnd=258 PartOfSpeech=VBG] [Text=... CharacterOffsetBegin=258 CharacterOffsetEnd=261 PartOfSpeech=:] [Text=the CharacterOffsetBegin=261 CharacterOffsetEnd=264 PartOfSpeech=DT] [Text=special CharacterOffsetBegin=265 CharacterOffsetEnd=272 PartOfSpeech=JJ] [Text=effects CharacterOffsetBegin=273 CharacterOffsetEnd=280 PartOfSpeech=NNS] [Text=... CharacterOffsetBegin=280 CharacterOffsetEnd=283 PartOfSpeech=:] [Text=ALL CharacterOffsetBegin=283 CharacterOffsetEnd=286 PartOfSpeech=NN] [Text=FALL CharacterOffsetBegin=287 CharacterOffsetEnd=291 PartOfSpeech=NN] [Text=FLAT CharacterOffsetBegin=292 CharacterOffsetEnd=296 PartOfSpeech=NN] [Text=!! CharacterOffsetBegin=296 CharacterOffsetEnd=298 PartOfSpeech=CD] 
root(ROOT-0, acting-17)
prep(acting-17, in-1)
pobj(in-1, attempt-2)
aux(keep-4, to-3)
infmod(attempt-2, keep-4)
det(group-7, a-5)
amod(group-7, mysterious-6)
dobj(keep-4, group-7)
prep(keep-4, from-8)
pcomp(from-8, stealing-9)
det(story-14, a-10)
amod(story-14, deadly-11)
nn(story-14, nerve-12)
nn(story-14, agent.The-13)
dobj(stealing-9, story-14)
punct(acting-17, ...-15)
det(acting-17, the-16)
punct(acting-17, ...-18)
det(effects-21, the-19)
amod(effects-21, special-20)
dobj(acting-17, effects-21)
punct(effects-21, ...-22)
nn(FLAT-25, ALL-23)
nn(FLAT-25, FALL-24)
dep(effects-21, FLAT-25)
num(FLAT-25, !!-26)

Sentence #3 (1 tokens):
!
[Text=! CharacterOffsetBegin=303 CharacterOffsetEnd=304 PartOfSpeech=.] 
root(ROOT-0, !-1)

Sentence #4 (5 tokens):
Definitely A MUST AVOID!!!
[Text=Definitely CharacterOffsetBegin=304 CharacterOffsetEnd=314 PartOfSpeech=RB] [Text=A CharacterOffsetBegin=315 CharacterOffsetEnd=316 PartOfSpeech=DT] [Text=MUST CharacterOffsetBegin=317 CharacterOffsetEnd=321 PartOfSpeech=JJ] [Text=AVOID CharacterOffsetBegin=322 CharacterOffsetEnd=327 PartOfSpeech=NN] [Text=!!! CharacterOffsetBegin=327 CharacterOffsetEnd=330 PartOfSpeech=CD] 
root(ROOT-0, AVOID-4)
advmod(AVOID-4, Definitely-1)
det(AVOID-4, A-2)
amod(AVOID-4, MUST-3)
num(AVOID-4, !!!-5)

Sentence #5 (1 tokens):
!
[Text=! CharacterOffsetBegin=335 CharacterOffsetEnd=336 PartOfSpeech=.] 
root(ROOT-0, !-1)

